docs/ingestion/data-formats.md

```diff
@@ -185,8 +185,8 @@ The ORC `inputFormat` has the following components:
 
 | Field | Type | Description | Required |
```

Comment:
Since you're editing this file can you also fix the json on line 177 please - it's missing a comma after the `}`

Reply:
I've made the edit, but am unclear if it'll make this PR.. Have added to my notes though to make sure it gets in at next opportunity. 

Thanks for the help with this, Suneet!